.eslintrc.json

```diff
@@ -0,0 +1,105 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true,
+        "node": true
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:react/recommended",
+        "plugin:import/errors",
+        "plugin:import/warnings"
+    ],
+    "parserOptions": {
```

Comment:
You may add the 'strict' configuration here.

Reply:
'[;;';'

Reply:
updated the rule set